src/pages/mesh/advanced/context-state.md

```diff
@@ -0,0 +1,410 @@
+
+Context state is eventually consistent. Generally, entries are available instantly within the edge they are created in. It may take 60 seconds or more for changes to propagate globally.
+
+You can also provide the following:
+
+- Time-to-live (TTL) - (Optional) provide a TTL in seconds for the entry. The TTL defaults to `604800` seconds (7 days). TTL values must be between `60` seconds and `604800` seconds (7 days).
+
+- Blocking - (Optional) provide a blocking flag for the operation. Defaults to `false` allowing mesh to proceed with code execution without waiting for a response. If blocking is `true`, the mesh will wait for a response, which allows you to ensure the operation was successful.
+
+<InlineAlert variant="info" slots="text"/>
+
+Setting the blocking flag to `true` can add latency to the overall query responses.
+
+- Put a simple entry:
+
+    ```typescript
+    context.state.put('example_key', 'example_value');
+    ```
+
+- Put a complex entry containing multiple serialized JSON values
+
+    ```typescript
+    context.state.put('example_key', '{"example_key_1": "example_value_1", "example_key_2": "example_value_2"}');
+    ```
+
+- Put a simple entry with 10 minute TTL. This means the entry will no longer be available after 10 minutes:
+
+    ```typescript
+    context.state.put('example_key', 'example_value', { ttl: 600 });
+    ```
+
+- Put a simple entry in a blocking manor when verification of the put is required:
+
+    ```typescript
+    context.state.put('example_key', 'example_value', { blocking: true });
+    ```
+
+#### Limitations
+
+When adding a key-value pair, you have the following limitations:
+
+- Key and Value type must be `string` or `binary`
+- Maximum key size of `512 bytes`
+- Maximum value size of `1 MB`
+
+### Get a value
+
+Get allows users to fetch a value for a given key.
+
+- Get a value:
+
+    ```typescript
+    context.state.get('example_key');
+    ```
+
+### Delete a key
+
+Delete allows users to delete a key and its associated value. A delete propagates to all edge locations.
+
+You can also provide the following:
+
+- Blocking - (Optional) a blocking flag for the operation. Defaults to `false` allowing mesh to proceed with code execution without waiting for a response. If blocking is `true`, the mesh will wait for a response, which allows you to ensure the delete was successful.
+
+<InlineAlert variant="info" slots="text"/>
+
+Setting the blocking flag to `true` can add latency to the overall query responses.
+
+- Delete a key:
+
+    ```typescript
+    context.state.delete('example_key');
+    ```
+
+- Delete a key in a blocking manor when verification of the delete is required:
+
+    ```typescript
+    context.state.delete('example_key', 'example_value', { blocking: true });
+    ```
+
+## Adobe Commerce as a Cloud Service (SaaS) REST example
+
+&#8203;<Edition name="saas" />
+
+**Prerequisites**
+
+- Add the **Adobe Commerce as a Cloud Service** API to your Developer Console workspace.
```

Comment:
Add links to the prereqs?

Reply:
@keharper can you clarify what prerequisites i should be linking to?

Reply:
Since Mesh isn't a Commerce entity, I'm not sure. Maybe [Projects and Workspaces](https://developer.adobe.com/graphql-mesh-gateway/mesh/basic/work-with-mesh/#projects-and-workspaces) needs to be expanded to list the required and optional developer console APIs. [Create a mesh](https://developer.adobe.com/graphql-mesh-gateway/mesh/basic/create-mesh/) never explicitly says select the **API Mesh* API/Service.

Reply:
good point @keharper. i've added a stopgap for now, but i plan to revisit the dev console guidance in the future to be more explicit and less reliant on the app builder docs.